### PR TITLE
Can call setPrivileged only once.

### PR DESCRIPTION
**Pull request by rbharath**
More details once ready

### CODE REVIEW

File: tests/markettokentest/burnable_functionality_test.go [hunk 1]
```diff
@@ -1,43 +1,55 @@
...
 	expectedBal := bal.Add(bal, big.NewInt(test.ONE_WEI*2)) // minted 4, burned 2
-	newBal, _ := deployed.MarketTokenContract.BalanceOf(nil, context.AuthUser3.From)
+	newBal, _ := marketTokenCont.BalanceOf(nil, context.AuthUser3.From)
 	if newBal.Cmp(expectedBal) != 0 {
 		t.Errorf("Expected new balance to be %v, got: %v", expectedBal, newBal)
 	}
 
 	// reset the actual listing privileged address
-	_, privErr2 := deployed.MarketTokenContract.SetPrivileged(test.GetTxOpts(context.AuthOwner, nil, big.NewInt(test.ONE_GWEI*2),
+	_, privErr2 := marketTokenCont.SetPrivileged(test.GetTxOpts(context.AuthOwner, nil, big.NewInt(test.ONE_GWEI*2),
```
Comment (robrobbins) [inline tests/markettokentest/burnable_functionality_test.go]: wut. we should not be able to reset these now right?

this should become a check for _can't change privileged address_

Comment (rbharath) [reply tests/markettokentest/burnable_functionality_test.go]: Good catch! Will update

File: tests/test/helpers.go [hunk 2]
```diff
@@ -35,6 +35,15 @@ func GenBytes32(str string) [32]byte {
...
 
+//GetCallOpts is a function which allows us to more succinctly place Call options
+func GetCallOpts(auth *bind.TransactOpts) *bind.CallOpts {
+	return &bind.CallOpts{
+		// TODO: Should this be a settable argument?
+		Pending: false,
+		From:    auth.From,
+	}
+}
+
```
Comment (robrobbins) [inline tests/test/helpers.go]: I don't mind this, but you can always just place `nil` as callopts for the simulated-backend. if you look around that's what i do.

File: tests/markettokentest/burnable_functionality_test.go [hunk 3]
```diff
@@ -48,8 +48,15 @@ func TestBurn(t *testing.T) {
...
 
-	// reset the actual listing privileged address
+	// Test that attempting to reset the listing privileged address doesn't work
 	_, privErr2 := marketTokenCont.SetPrivileged(test.GetTxOpts(context.AuthOwner, nil, big.NewInt(test.ONE_GWEI*2),
 		100000), deployed.ListingAddress, deployed.ReserveAddress)
-	test.IfNotNil(t, privErr2, "Error resetting Market Token privileged contracts")
+	test.IfNotNil(t, privErr2, "Error attempting to reset Market Token privileged contracts")
+
+	// Check that the listing address hasn't changed
+	actualListingAddress, _, getErr := marketTokenCont.GetPrivileged(test.GetCallOpts(context.AuthOwner))
```
Comment (robrobbins) [inline tests/markettokentest/burnable_functionality_test.go]: as stated, you can just pass `nil` to any evm 'getter' (in our specs) vs an actual callOpts. this is ok tho, we can leave it

**Pull request merged**